README.md

```diff
@@ -39,6 +39,9 @@ info(io): opening "0_0.tigerbeetle"...
 info(main): 0: cluster=0: listening on 0.0.0.0:3000
 ```
 
+Note: if you are on macOS, you will need to call the Docker run
+command with `--cap-add IPC_LOCK` or `--ulimit memlock=-1:-1`.
```

Comment:
Add a "for more detail ..." link to the corresponding section of our docker setup docs (or the github issue discussion, whatever you think is most useful/relevant).

Reply:
Done, thanks!